cmd/fleetctl/get.go

```diff
@@ -1022,3 +1055,55 @@ func getSoftwareCommand() *cli.Command {
 		},
 	}
 }
+
+func getMDMAppleCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "mdm_apple",
+		Hidden:  true, // TODO: temporary, until the MDM feature is officially released
+		Aliases: []string{"mdm-apple"},
+		Usage:   "Show Apple Push Notification Service (APNs) information",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
+		Action: func(c *cli.Context) error {
+			const expirationWarning = 30 * 24 * time.Hour // 30 days
+
+			client, err := clientFromCLI(c)
+			if err != nil {
+				return err
+			}
+
+			mdm, err := client.GetAppleMDM()
+			if err != nil {
+				var nfe service.NotFoundErr
+				if errors.As(err, &nfe) {
+					log(c, "Error: No Apple Push Notification service (APNs) certificate found. Use `fleetctl generate mdm-apple` and then `fleet serve` with `mdm` configuration to turn on MDM features.\n")
+					return nil
+				}
+				return err
+			}
+
+			printKeyValueTable(c, [][]string{
+				{"Common name (CN):", mdm.CommonName},
+				{"Serial number:", mdm.SerialNumber},
+				{"Issuer:", mdm.Issuer},
+				{"Renew date:", mdm.RenewDate.Format("January 2, 2006")},
+			})
+
+			warnDate := time.Now().Add(expirationWarning)
+			if mdm.RenewDate.Before(time.Now()) {
+				// certificate is expired, print an error
+				log(c, "\n")
+				printColoredMsg(c, "ERROR: Your Apple Push Notification service (APNs) certificate is expired. MDM features are turned off. To renew your APNs certificate, follow these instructions: [TODO link to documentation]", tablewriter.FgHiRedColor)
+			} else if mdm.RenewDate.Before(warnDate) {
+				// certificate will soon expire, print a warning
+				log(c, "\n")
+				printColoredMsg(c, "WARNING: Your Apple Push Notification service (APNs) certificate is less than 30 days from expiration. If it expires, MDM features will be turned off. To renew your APNs certificate, follow these instructions: [TODO link to documentation]", tablewriter.FgHiYellowColor)
```

Comment:
@noahtalerman adding for reference to update when is https://github.com/fleetdm/fleet/issues/8799 completed.

Reply:
Thanks, this is noted in #8799:

> Link to docs in the output of fleetctl get mdm-apple command.

Will mark as resolved.